AdminSetPassword.php

```diff
@@ -16,7 +16,7 @@ public function register()
     {
         $this->name = 'Admin Set Password';
         $this->description = 'Displays an input field and button on "manage_user_edit_page" to change a users password directly.';
```

Comment:
These 2 strings should also be translated

Reply:
Added since last commit.

Reply:
I mean the sub directory `js` is more transparency. In larger projects I leave all assets in `assets/js` or `css` and more.